code/src/components/Intro.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const Popcorn = () => (
```

Comment:
Try to be consistent with the naming, so that the main function and file are named the same